.travis.yml

```diff
@@ -1,5 +1,15 @@
 language: ruby
-before_install: gem install bundler -v '1.17.3'
```

Comment:
My experiments suggest that if you add
```
dist: xenial
```
and then drop all the rmagick specific install steps, that this will just work.